src/brc/conf/api.py

```diff
@@ -25,5 +25,9 @@
     },
 
     # no geo things here
-    'DEFAULT_FIELD_INSPECTORS': BASE_SWAGGER_SETTINGS['DEFAULT_FIELD_INSPECTORS'][1:],
+    'DEFAULT_FIELD_INSPECTORS': (
+        'zds_schema.inspectors.files.FileFieldInspector',
```

Comment:
do we use `FileField` in BRC?

Reply:
yes!.... ok no, but i practiced copy/paste skills :>